python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -1666,6 +1676,9 @@ def _makeParSchema(self, parInfo, pars, parSuperStarFlat,
         parSchema.addField('mirrorChromaticityPivot', type='ArrayD',
                            doc='Mirror chromaticity pivot mjd',
                            size=pars['MIRRORCHROMATICITYPIVOT'].size)
+        parSchema.addField('compCcdChromaticity', type='ArrayD',
```

Comment:
As above, you use capitals for the `compCCDChromaticity` variable name in other files, e.g. https://github.com/search?q=org%3Alsst+compCcdChromaticity&type=code.  I prefer the camelCase used here...but consistency is good (and is it not necessary?!) 😉

Reply:
This has DM-style camel case because it's a DM persisted product.